sessions-core/src/main/java/com/adyen/checkout/sessions/model/setup/Configuration.kt

```diff
@@ -0,0 +1,38 @@
+package com.adyen.checkout.sessions.model.setup
+
+import com.adyen.checkout.core.exception.ModelSerializationException
+import com.adyen.checkout.core.model.ModelObject
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+@Parcelize
+data class Configuration(val enableStoreDetails: Boolean? = null) : ModelObject() {
```

Comment:
A more descriptive name would be nice as we already have 2 classes with this name. Maybe something like `SessionSetupConfiguration`